docs/docs/plugins/getting-started.md

```diff
@@ -0,0 +1,17 @@
+# Getting Started
+
+## Quick Start
+
+Use the "Web Retriever" plugin by adding this url:
+https://web-retriever-draganjovanovich.vercel.app/ai-plugin.json
```

Comment:
see https://github.com/LAION-AI/Open-Assistant/pull/3277

Reply:
Yeah - unsure if should just merge this PR and then just do small PR to update the docs again when FE ready?

Reply:
will update the url to be the OA hosted one for sure as soon as its available

Reply:
I think we should only include OA-provided plugins in this doc, otherwise we will probably end up with many PRs of everyone trying to advertise their own plugins. Web retriever will be merged soon (probably today) so we can edit this and then merge after

Reply:
Yeah was wondering, although could be nice to just maintain a list of community plugins below the OA ones and then just clearly differentiate between the two and add lots of disclaimers that the community ones are unofficial. 

We could only add decent ones then that the community itself has found useful. Would be nice to have them discoverable in some way. Might help encourage bit of an ecosystem around them too.

I could rejig the PR to so sort of what I mean about distinguishing.

Reply:
Influx does a good job on this with telegraf plugins for example.

https://docs.influxdata.com/telegraf/v1.26/plugins/

Obviously we don't need to overdo any of this either since probably be small enough list to start with.

Reply:
example of what i mean in latest commit to this pr 
![image](https://github.com/LAION-AI/Open-Assistant/assets/2178292/acd7e280-5149-44e4-be35-d04fd971bae5)